gaupol/agents/video.py

```diff
@@ -27,7 +27,11 @@
 from gi.repository import Gtk
 
 with aeidon.util.silent(Exception):
+    from gi import require_version
+    require_version('GstTag', '1.0')
```

Comment:
Please move the version requirement to `gaupol/__init__.py`. It's better to have those in one place as the modules can be imported and used in many places.